docs/proc/code_contribution_guide.md

```diff
@@ -0,0 +1,31 @@
+# Bref
+As team we should follow rules to have understandable processes and code. This document covers how we work with versions, codes and its styles. If repository does not contain own rules, then this rules SHOULD be used.
+
+Also please pay attention that some repositories has more complicated rules, these rules have own sections inside this document.
+
+# Code style guide
+
+- Rust: we respect [rust code style guide](https://doc.rust-lang.org/nightly/style-guide/) and we should follow it. Exception: if repository has own code style guide (ex: [polkadot-sdk](https://github.com/paritytech/substrate/blob/master/docs/STYLE_GUIDE.md)) we should use it.
```

Comment:
Should we include tools like [`clippy`](https://doc.rust-lang.org/clippy/index.html), [`miri`](https://github.com/rust-lang/miri) or something else non-formatting related in this guide?

Reply:
Add in: c41730b0024b08671cdf60d43d8f7d35003ad378